tests/ui/lint/default-groups-issue-65464.rs

```diff
@@ -0,0 +1,14 @@
+//@ check-pass
+
+// Verify information about membership to builtin lint group is included in the lint message when
+// explaining lint level and source for builtin lints with default settings.
+//
+// Ideally, we'd like to use lints that are part of `unused` group as shown in the issue.
+// This is not possible in an ui test, because `unused` lints are enabled with `-A unused`
+// in such tests, and the we're testing a scenario with no modification to the default settings.
```

Comment:
```suggestion
// in such tests, and we want to test a scenario with no modification of the default settings.
```
Do we want that, or are the default settings the only option here? To be honest, I wonder if this test is necessary at all, given the coverage provided by all the existing tests.

Reply:
Hmm you may be right, the test in current state doesn't actually test anything better than rest of the tests that capture lint output.